about.md

```diff
@@ -10,15 +10,15 @@ FOSSASIA 2014 is an exciting Open Technology event for developers, students, and
 
 Since the first FOSSASIA meetup in 2009 in Ho Chi Minh City, FOSSASIA has taken the lead to organize Free and Open Source Software conferences in Vietnam. Past FOSSASIA events have attracted over 1400 participants and speakers from 24 countries, with more than 100 volunteers. More than half of all participants were women.
 
-FOSSASIA is an organization developing software applications for social change using a wide-range of technologies. It was established 2009. Projects range from Free and Open Source software, to design, graphics and hardware. We also organize and participate in conferences, meetups and code camps. The annual FOSSASIA OpenTechSummit is one of the largest gatherings of developers in Asia and took place in Vietnam, Cambodia and Singapore. Please join us and start contributing to our projects on [GitHub](https://github.com/fossasi.org).
+FOSSASIA is an organization developing software applications for social change using a wide-range of technologies. It was established in 2009. Projects range from Free and Open Source software, to design, graphics and hardware. We also organize and participate in conferences, meetups and code camps. The annual FOSSASIA OpenTechSummit is one of the largest gatherings of developers in Asia and took place in Vietnam, Cambodia and Singapore. Please join us and start contributing to our projects on [GitHub](https://github.com/fossasi.org).
 
 ##FOSSASIA Activities
 
 FOSSASIA works together with Open Source projects in Asia and around the world and develops Free and Open Source software for social change.
 
 We bring developers, designers and start ups together, provide infrastructure and organize code sprints, developer meet ups and community gatherings.
 
-Every year Fossasia participates in events like [Google Code](https://codein.withgoogle.com) in and [Google summer of Code](https://summerofcode.withgoogle.com/), allowing young developers to contribute to open-source. Each year Fossasia becomes the recipient of the most participants, and collectively they do 100s of tasks, helping Fossasia and at the same time improving as developers.
+Every year Fossasia participates in events like [Google Code In](https://codein.withgoogle.com) in and [Google summer of Code](https://summerofcode.withgoogle.com/), allowing young developers to contribute to open-source. Each year Fossasia becomes the recipient of the most participants, and collectively they do 100s of tasks, helping Fossasia and at the same time improving as developers.
```

Comment:
Delete the `in` after `Google Code In`
and change `Google Code In` to ` Google Code-In`